packages/vue/src/plugins/format/FormatPlugin.ts

```diff
@@ -0,0 +1,75 @@
+import { DateFormat, FDate } from "@fkui/date";
+import { formatNumber, parseBankgiro, parseDate } from "@fkui/logic";
+import { type App, type DirectiveBinding, type Plugin } from "vue";
+
+interface DateRange {
+    from: string;
+    to: string;
+}
+
+function formatNumbers(el: HTMLElement, number: string | number): void {
```

Comment:
Här borde vi nog ta `unknown` för konsumenten kan ju faktiskt skicka in precis vad som helst (allt från `null` och `undefined` till klasser osv) och själva kolla typen. Är det något orimligt kan vi sätta en tom text.